tests/macros/tmacros_various.nim

```diff
@@ -194,3 +194,41 @@ static:
     doAssert(v == a)
 
   echo "macrocache ok"
+  
+block tupleNewLitTests:
+  echo "tupleNewLitTests"
```

Comment:
@solo989 the tests fail because you're adding `echo` and you haven't updated the `nimout` section on top; I recommend simply removing the `echo` statements you've added; it serves no purpose (doAssert would show the context on error)